content/en/containers/kubernetes/appsec.md

```diff
@@ -0,0 +1,360 @@
+- Manually configure specific gateways
+- Temporarily disable Appsec for testing
+- Exclude certain gateways from security monitoring
+
+**Note**: By default, all resources are included. Only resources with the label explicitly set to `"false"` are excluded.
+
+## Troubleshooting
+
+### Injector not detecting proxies
+
+**Symptom**: No `EnvoyExtensionPolicy` or `EnvoyFilter` resources are created.
+
+**Solutions**:
+- Check that `autoDetect` is set to `true` or proxies are manually specified
+- Verify the Cluster Agent logs for proxy detection messages
+- Ensure your proxies are installed and have the expected Kubernetes resources (Gateway, GatewayClass)
+- Try manually specifying proxy types using the `proxies` parameter
+
+### EnvoyExtensionPolicy or EnvoyFilter not created
+
+**Symptom**: Injector is running but configuration resources are missing.
+
+**Solutions**:
+- Check Cluster Agent logs for RBAC permission errors
+- Verify the Cluster Agent service account has permissions to create `EnvoyExtensionPolicy` or `EnvoyFilter` resources
+- Ensure the processor service exists and is accessible
+- Check for conflicting existing policies or filters
+
+### Traffic not being processed
+
+**Symptom**: No security events appear in the Datadog UI.
+
+**Solutions**:
+- Verify the external processor deployment is running: `kubectl get pods -n datadog -l app=datadog-aap-extproc`
+- Look for warning logs in your reverse proxies concerning this part of the configuration.
+- Check processor logs for connection errors: `kubectl logs -n datadog -l app=datadog-aap-extproc`
+- Verify the processor service is correctly configured and resolvable
+- Test connectivity from gateway pods to the processor service
+- Ensure [Remote Configuration][4] is enabled in your Datadog Agent
+
+### External processor connection issues
+
+**Symptom**: Gateways cannot reach the external processor.
+
+**Solutions**:
+- Verify the processor service name and namespace match your configuration
+- Check for NetworkPolicy rules blocking cross-namespace traffic
+- For Envoy Gateway: Ensure `ReferenceGrant` resources exist for cross-namespace service references
+- Test DNS resolution from gateway pods: `nslookup datadog-aap-extproc-service.datadog.svc.cluster.local`
+- Verify the processor port configuration matches the service definition
+
+### RBAC permission errors
+
+**Symptom**: Cluster Agent logs show permission denied errors.
+
+**Solutions**:
+- Verify the Cluster Agent ClusterRole includes permissions for:
+  - `gateway.envoyproxy.io/envoyextensionpolicies` (create, update, delete, get, list, watch)
+  - `networking.istio.io/envoyfilters` (create, update, delete, get, list, watch)
+  - `gateway.networking.k8s.io/gateways` (get, list, watch)
+  - `gateway.networking.k8s.io/gatewayclasses` (get, list, watch)
+- Check that the ClusterRoleBinding references the correct service account
+
+## Limitations
```

Comment:
I would consider putting this section at the top of the doc so that users that do not meet the req's don't learn about that after trying to set this up. I'll add it to the top. If you agree, you can review it from here.